build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/InternalJavaRestTestPlugin.java

```diff
@@ -41,12 +41,7 @@ public void apply(Project project) {
 
         // setup the javaRestTest task
         // we use a StandloneRestIntegTestTask here so that the conventions of RestTestBasePlugin don't create a test cluster
-        TaskProvider<StandaloneRestIntegTestTask> testTask = registerTestTask(
-            project,
-            javaTestSourceSet,
-            SOURCE_SET_NAME,
-            StandaloneRestIntegTestTask.class
```

Comment:
I didn't saw any conventions in RestTestBasePlugin anymore. Replaced StandaloneRestIntegTestTask by RestIntegTestTask to make things easier.

Reply:
I'm increasingly confused by the distinction between these two types.

Reply:
Same.